bin/create

```diff
@@ -33,12 +33,12 @@ if (args['--help'] || args._.length === 0) {
     console.log('    --xwalk-apk-url configure the download URL of the Crosswalk runtime library.');
     process.exit(1);
 }
-if (args['--xwalk-shared-library'] || args['--xwalk-apk-url']) {
+if (args['--xwalk-shared-library'] || args['--xwalk-apk-url'] != undefined) {
     if (args['--shared']) {
         console.log('    --shared do not support --xwalk-shared-library and --xwalk-apk-url.');
```

Comment:
Just a reminder that this needs to be fixed at some point (either here or in another PR: "do" should be "does").